README.md

```diff
@@ -45,7 +45,9 @@ The onboarding path for Anserini starts [here](docs/start-here.md)!
 <details>
 <summary>Windows tips</summary>
 
-Note that on Windows, tests may fail due to encoding issues, see [#1466](https://github.com/castorini/anserini/issues/1466).
+If you are using Windows, please use WSL2 to build Anserini. Please refer to the [WSL2 Installation](https://learn.microsoft.com/en-us/windows/wsl/install) document to install WSL2 if you haven't already.
```

Comment:
we use the convention of a sentence on each line, to make the diffs more easily readable.